app/javascript/packages/document-capture/components/in-person-troubleshooting-options.tsx

```diff
@@ -0,0 +1,53 @@
+import { useContext } from 'react';
+import { TroubleshootingOptions } from '@18f/identity-components';
+import { useI18n } from '@18f/identity-react-i18n';
+import type { TroubleshootingOption } from '@18f/identity-components/troubleshooting-options';
+import ServiceProviderContext from '../context/service-provider';
+import HelpCenterContext from '../context/help-center';
+
+interface InPersonTroubleshootingOptionsProps {
+  /**
+   * Custom heading to show in place of default.
+   */
+  heading?: string;
+
+  /**
+   * Location parameter to append to links.
+   */
+  location?: string;
+}
+
+function InPersonTroubleshootingOptions({
+  heading,
+  location = 'in_person_troubleshooting_options',
+}: InPersonTroubleshootingOptionsProps) {
+  const { t } = useI18n();
+  const { getHelpCenterURL } = useContext(HelpCenterContext);
+  const { name: spName, getFailureToProofURL } = useContext(ServiceProviderContext);
+
+  return (
+    <TroubleshootingOptions
+      heading={heading}
+      options={
+        [
+          {
+            url: getHelpCenterURL({
+              category: 'verify-your-identity',
+              article: 'how-to-verify-in-person',
+              location,
+            }),
```

Comment:
Dunno how best we want to track it, but noting that this URL currently redirects back to the account page since it's not included in the allowlist:

https://github.com/18F/identity-idp/blob/5a5e738bca8c75ad07dd3508cfbe6897af3b2d82/app/services/marketing_site.rb#L6-L15

Reply:
I added the page to the allow list in a61875d901806d4f04f514cf33bab0c1089b1591 and wrote ticket LG-6875 to make sure we don't forget to update it once the actual article is published.